docs/development.md

```diff
@@ -12,21 +12,29 @@ Interested in contributing to Masonry? Awesome! Take a look at our [contribution
     go get github.com/opencontrol/compliance-masonry
     ```
 
-1. Install the dependencies.
+1. Run the tool.
 
     ```sh
-    cd $GOPATH/src/github.com/opencontrol/compliance-masonry
-    dep ensure
-    go install .
+    compliance-masonry
     ```
 
-1. Run the tool.
+This should print out usage documentation.
+
+## Updating dependencies
+
+1. Get the `vndr` handling tool.
 
     ```sh
-    compliance-masonry
+    go get github.com/LK4D4/vndr
     ```
 
-This should print out usage documentation.
+1. Update dependencies by running the `vndr` tool in the project.
```

Comment:
"in the project" eludes to the gopath, but should call it out...

$GOPATH/src/github.com/opencontrol/compliance-masonry/